rotkehlchen/tests/unit/test_optimism_transactions.py

```diff
@@ -58,3 +68,56 @@ def assert_tx_okay(transactions, should_have_l1):
     with optimism_transactions.database.conn.read_ctx() as cursor:
         tx, _ = optimism_transactions.get_or_create_transaction(cursor, tx_hash, string_to_evm_address(to_address))  # noqa: E501
         assert_tx_okay([tx], should_have_l1=True)
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('optimism_accounts', [['0xc37b40ABdB939635068d3c5f13E7faF686F03B65']])
+@pytest.mark.parametrize('optimism_manager_connect_at_start', [(
+    WeightedNode(node_info=NodeName(name='drpc', endpoint='https://optimism.drpc.org', owned=False, blockchain=SupportedBlockchain.OPTIMISM), active=True, weight=FVal('0.5')),  # noqa: E501
+    WeightedNode(node_info=NodeName(name='mainnet', endpoint='https://mainnet.optimism.io', owned=False, blockchain=SupportedBlockchain.OPTIMISM), active=True, weight=FVal('0.5')),  # noqa: E501
+)])
+def test_l1_fee_queried_when_missing(
+        optimism_transactions: 'OptimismTransactions',
+        optimism_accounts: list['ChecksumEvmAddress'],
+        optimism_manager_connect_at_start: Sequence[WeightedNode],
```

Comment:
can you also check for a transaction after bedrock?

Reply:
can you maybe mention that in the docstring of that test?